docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+
+The spec is maintained by .NET runtime implementors, specifically Microsoft (includes .NET Framework, .NET Core and Mono) and Unity. A public feedback process is used as part of establishing new .NET Standard Library versions.
+
+Official Artifacts
+------------------
+
+The official spec is a set of .cs files that define the APIs that are part of the standard. The [ref directory](https://github.com/dotnet/corefx/tree/master/src/System.Runtime/ref) for each [component](https://github.com/dotnet/corefx/tree/master/src) defines the .NET Standard Library APIs. While the ref artifacts reside in the [CoreFX repo](https://github.com/dotnet/corefx), they are not .NET Core specific.
+
+The [NETStandard.Library](https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets) metapackage describes the set of libraries that define (in part) one or more .NET Standard Library versions.
+
+Derivative artifacts are provided to enable more convenient reading and to enable certain developer scenarios (for example, using a compiler).
+
+- API list in markdown (TBD)
+- Reference assemblies, distributed as [NuGet packages](../core-concepts/packages.md) and referenced by the [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library/) metapackage.
+
+Package Representation
+----------------------
+
+The primary distribution vehicle for the .NET Standard Library reference assemblies is [NuGet packages](../core-concepts/packages.md). Implementations will be delivered in a variety of ways, appropriate for each .NET runtime.
+
+NuGet packages target one of more frameworks. The .NET Standard Library packages target the ".NET Standard" framework, which is represented by the the `netstandard` Target Framework Moniker (TFM). The framework is ".NET Standard" instead of ".NET Standard Library", since ".NET Standard" includes both the .NET Standard Library packages and the larger ecosystem of libraries that are built on top of it. Libraries that are intended to run on multiple runtimes should use this target. 
+
+The `NETStandard.Library` metapackage references the complete set of NuGet packages that define the .NET Standard Library.  The most common way to target `netstandard` is by referencing this metapackage. It describes and provides access to the ~40 .NET  libraries and associated APIs that define the .NET Standard Library. You can reference additional packages that target `netstandard` to get access to additional APIs. 
+
+The `NETStandard.Library` metapackage doesn't define a framework of its own, but exposes the set of frameworks defined by the packages it references, including `netstandard`. Similarly, the `netstandard` framework does not expose any APIs on its own, but relies on packages that expose `netstandard` assets to define the framework. The definition of the `netstandard` framework is not limited to the .NET Standard Library.
```

Comment:
This was confusing for me to read.  

Correct me if I'm wrong, but based on behavior when restoring various `netstandard1.x` versions, I've seen the relationship thusly:

* `NETStandard.Library` defines a set of packages
* The latest version of `netstandard1.x` pulls down every single package
* Lower versions of `netstandard1.x` pull down a subset of that set of packages
* The rules for how that subset of packages is defines is ....magic I guess?
* You can pull in any other package so long as its `netstandard1.x` version is less than or equal to your own

It doesn't seem like something that can be boiled down to a single paragraph without losing critical information.